manifests/postgres-operator.yaml

```diff
@@ -269,7 +269,7 @@ spec:
       serviceAccountName: postgres-operator
       containers:
         - name: postgres-operator
-          image: docker.io/flanksource/postgres-operator:{{.postgresOperator.version}}
+          image: flanksource/postgres-operator:{{.postgresOperator.version}}
```

Comment:
```suggestion
          image: docker.io/flanksource/postgres-operator:{{.postgresOperator.version | default "na" }}
```

Reply:
karina defaults in an operator version if not supplied (I bumped to 1.6.2 in this change set) - would it not be better to have the same value here?